calendar.rb

```diff
@@ -0,0 +1,98 @@
+require './event.rb'
+class Calendar
+  def initialize
+    @events = {}
+  end  
```

Comment:
there are extra spaces at the end of this line. make sure to have no extra spaces on line ends.